.gitignore

```
@@ -1 +1,3 @@
 /.idea
+/node_modules
+/.vscode
```

```suggestion
/node_modules
```

In my opinion those 2 shouldn't be in the gitignore of your project, but in your global gitignore. Otherwise everyone with another editor would have to edit every project.

---

I wasn't even aware there was an option for a global gitignore and I've been using git for 10+ years now 😅 You learn something new every day.

If there's a possibility that adding the IDE config folders to the gitignore can help someone ignorant (like me) in the future I see no harm in including them.